### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

File: .github/workflows/gradle-build-pr.yml [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+  pull_request:
+    paths:
+      - !.github/workflows/devimageupadte.yaml
 jobs:
   deploy_to_integration:
     timeout-minutes: 180
     strategy:
       matrix:
         os: [ubuntu-latest]
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
```
Comment (smark88) [inline .github/workflows/gradle-build-pr.yml]: this doesn't work on PR nor does the previous syntax

File: .github/workflows/gradle-build-pr.yml [hunk 2]
```diff
@@ -1,14 +1,17 @@
 name: Run Gradle on PRs
-on: pull_request
+on:
+  pull_request:
+    paths:
+      - !.github/workflows/devimageupadte.yaml
```
Comment (fboulnois) [inline .github/workflows/gradle-build-pr.yml]: also rename file

```suggestion
      - !.github/workflows/dev-image-update.yaml
```

**Review by fboulnois — CHANGES_REQUESTED** [review]
Should we create a GitHub Action template that both workflows can use? Otherwise, see other suggestions.

**Comment by smark88** [comment]
> Should we create a GitHub Action template that both workflows can use? Otherwise, see other suggestions.

all jobs run on separate runners even if in the same yaml.

File: .github/workflows/gradle-build-pr.yml (hunk 1 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml]: found it 
```
if: "!contains( github.event.pull_request.labels.*.name, 'skip-ci')"
```

File: .github/workflows/dev-image-update.yaml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+          gcloud container images add-tag gcr.io/${DEV_PROJECT}/jade-data-repo:${GCR_TAG} \
+            gcr.io/${DEV_PROJECT}/jade-data-repo:${GCR_TAG}-develop --quiet
+      - name: Find and replace image with current develop commit
+        run: |
+          find . -name devDeployment.yaml -type f -exec sh -c 'yq w -i $1 'datarepo-api.image.tag' $2' sh {} ${GCR_TAG} ';'
+      - name: Git add, commit and push
+        run: |
+          cd datarepo-helm-definitions
+          git config --global user.email "smark@broadinstitute.org"
+          git config --global user.name "smark88"
```
Comment (thathert) [inline .github/workflows/dev-image-update.yaml]: Can we switch this to something like robot@jade.team and robot?

**Review by thathert — CHANGES_REQUESTED** [review]
It would be nice if we could DRY this up like @fboulnois suggests above.

File: .github/workflows/dev-image-update.yaml (hunk 3 above)
Comment (smark88) [reply .github/workflows/dev-image-update.yaml]: It's tied to a personal developer key will that work ?

**Comment by smark88** [comment]
> It would be nice if we could DRY this up like @fboulnois suggests above.

I don't where to template in 

> It would be nice if we could DRY this up like @fboulnois suggests above.

I dont know where to template or dry in .github/workflows/dev-image-update.yaml

**Review by thathert — APPROVED** [review]
LGTM

**Review by fboulnois — CHANGES_REQUESTED** [review]
nvm

**Pull request merged**